Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/LayerTreeHost.cpp

```diff
@@ -433,6 +467,8 @@ void LayerTreeHost::didRenderFrameTimerFired()
 
 void LayerTreeHost::displayDidRefresh(PlatformDisplayID displayID)
 {
+    ASSERT(m_compositor->compositingRunLoop()->isCurrent());
+
     WebProcess::singleton().eventDispatcher().notifyScrollingTreesDisplayDidRefresh(displayID);
 }
```

Comment:
I think this is only called for non display link case, it would be good to add the ifdef to make it clearer.

Reply:
I'll add it, thanks!